site/frontend/src/pages/compare/compile/table/benchmark-detail.vue

```diff
@@ -210,7 +217,11 @@ onMounted(() => renderGraph());
       <div class="title">
         <div class="bold">{{ getGraphTitle() }}</div>
         <div style="font-size: 0.8em">
-          Each plotted value is relative to its previous commit
+          Each plotted value is relative to its previous commit.
+        </div>
+        <div style="font-size: 0.8em">
+          A shaded region shows values that are more recent than the profiled
```

Comment:
nit: maybe benchmarked instead of profiled? But no strong objections to keeping profiled.

Reply:
Sure.